src/components/dialogs/field-hook.js

```diff
@@ -419,19 +534,93 @@ export const useEquipmentTableValues = ({
                         )}
                     </Droppable>
                 </DragDropContext>
+                <Grid container>
+                    <Grid item xs justifyContent={'flex-end'} spacing={2}>
+                        <IconButton
+                            className={classes.iconColor}
+                            onClick={() => setOpenCSVImportDialog(true)}
+                        >
+                            <Tooltip
+                                title={intl.formatMessage({ id: 'ImportCSV' })}
+                                placement="bottom"
+                            >
+                                <Upload />
+                            </Tooltip>
+                        </IconButton>
+                    </Grid>
+                    <Grid item justifyContent={'flex-end'}>
+                        <IconButton
+                            className={classes.iconColor}
+                            onClick={() => handleAddValue()}
+                        >
+                            <AddIcon />
+                        </IconButton>
+
+                        <IconButton
+                            className={classes.iconColor}
+                            onClick={() => handleDeleteItem()}
+                            disabled={selectedIds.size === 0}
+                        >
+                            <DeleteIcon />
+                        </IconButton>
+                        <IconButton
+                            key={id + name + 'upButton'}
+                            disabled={
+                                selectedIds.size === 0 || selectedIds.has(0)
+                            }
+                            onClick={() => {
+                                handleChangeOrder(-1);
+                            }}
+                            className={classes.iconColor}
+                        >
+                            <ArrowCircleUp />
+                        </IconButton>
+                        <IconButton
+                            key={id + name + 'downButton'}
+                            disabled={
+                                selectedIds.size === 0 ||
+                                selectedIds.has(values.length - 1)
+                            }
+                            onClick={() => {
+                                handleChangeOrder(id, 1);
```

Comment:
here we passe 2 args to handleChangeOrder(), "down" and 1
While it accepts only one.
Strange

Reply:
ok i fix it to one argument